### DISCUSSION TITLE
Minimum ressources on VPS for install modoboa

### DISCUSSION THREAD

**Discussion by GFRT-tech** [modoboa/modoboa#2139] [General]
Hello,

What are the minimum resources required to run a modoboa server?

I tested on a HETZNER CENTOS 8 instance the latency is huge.
The whole thing became unusable because the CPU resources occupied were at 90%.

**Comment by MrGeneration** [comment]
Heya,

I think that depends on the number of users you wanna go with.
I've been using CX21 (2 vCPUs + 4GB Ram) for a very long time.
 
The moment `Redis` came on top of that I switched to CX31 (2 vCPU + 8GB Ram).
On both environments I neve really had any issues, but I also fiddled at some configuration parts that did not fit for me.

About a month ago I defragmented my structure and moved it onto my Proxmox Node, performance is not notably better or worse.

Edit:
To help you further.
I chose above sizing because of requirements of Modoboas dependencies and was expecting up to 6GB memory usage.
To be fair, my system only handles 10 *concurrent* users maximum which is why my system may be oversized.

I think a memory graph of the last 30 days may be useful.
![image](https://user-images.githubusercontent.com/6549061/105632259-7ea4f080-5e52-11eb-853b-085f3630a8d0.png)
 

Hope that helps.
